web/docusaurus.config.js

```diff
@@ -102,19 +102,19 @@ module.exports = {
           items: [
             {
               label: 'Blog',
-              to: 'https://supabase.io/blog',
+              to: '/blog',
```

Comment:
will this work? I think because it's a SPA it might need the full URL? (i think it's why the `/oss` link is failing below

Reply:
the /oss failing was a rewrite - it was missing rewrite config.

Reply:
i'll check in the preview urls

Reply:
yea your right it's goosed. will change back